### PR TITLE
Feature 25

### PR DESCRIPTION
**Pull request by jferdi24**
## Checklist ✅

- [ ] I have followed the [contributor guideline](https://github.com/platzily/platzily-ui/blob/master/CONTRIBUTING.md)
- [ ] Tests are included
- [ ] Documentation is changed or added
- [x] Related issue has been created
- [ ] Commits messages follows [commit guideline](https://github.com/platzily/platzily-ui/blob/master/CONTRIBUTING.md/#Commits)

resolves [Issue ID]

### CODE REVIEW

**Review by AbejaCruz — APPROVED** [review]

File: packages/api-gw/src/index.ts [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
-import { MongoConnection } from '../../db-connect-test/src/index';
-import { User } from "../../db-connect-test/src/User";
-
-const dbMongo = new MongoConnection('mongodb://localhost/test?retryWrites=true', [User]);
+import { User } from "../../service-users/src/entities/User";
 
 server.get('/users', async (req, res) => {
-  dbMongo.getConnection().then(async function (connect) {
-    const users = await connect.mongoManager.find(User);
+  MongoDb.getConnection().then(async function (connect) {
```
Comment (cesarp04) [inline packages/api-gw/src/index.ts:9]: in this package, there should not be a connection to the database

**Review by cesarp04 — CHANGES_REQUESTED** [review]
review the api-gw package comment and correct the file

File: packages/api-gw/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,75 @@
+import Fastify, { FastifyInstance } from 'fastify';
+import { MongoDb } from "../../service-db/src/index";
+
+const server: FastifyInstance = Fastify({})
+
+import { User } from "../../service-users/src/entities/User";
+
+server.get('/users', async (req, res) => {
+  MongoDb.getConnection().then(async function (connect) {
```
Comment (zago-code) [inline packages/api-gw/src/index.ts:9]: Diego, just to understand. This code would go in each service, be it, Users, links or campaigns. Here in api-gw, there is only the logic that would bring the data that each user service, link service, or campaign service has previously requested to service db. It's right?

Comment (jferdi24) [reply packages/api-gw/src/index.ts:9]: Yes. it's a example code

**Review by cesarp04 — APPROVED** [review]

**Pull request merged**